added equilibira + some kinetics topics notes

Samgonza opened the pull request:
Added some quick notes to the equilibria notes and kinetics subtopics, I'm not sure which topics should be included be highlighted or included. I could also add in steady state approximations or enzymes?

Samgonza committed:
added equilibira + some kientcis topics notes

cadalyjr commented:
steady state approximations or enzymes are good ideas!

cadalyjr reviewed (changes requested):
minor change - "No changes are observed at equilibrium." should be "No **overall** changes are observed at equilibrium."

Samgonza committed:
Update chem.md

Changed "No changes at equilibrium " to "No overall changes are observed at equilibrium"

Samgonza commented:
> minor change - "No changes are observed at equilibrium." should be "No **overall** changes are observed at equilibrium."

I made these changes to the file
